pkg/asset/installconfig/kubevirt/validation_test.go

```diff
@@ -69,6 +71,8 @@ func TestKubevirtInstallConfigValidation(t *testing.T) {
 			expectClient: func(kubevirtClient *mock.MockClient) {
 				kubevirtClient.EXPECT().GetNetworkAttachmentDefinition(gomock.Any(), validNetworkName, validNamespace).Return(nil, nil).AnyTimes()
 				kubevirtClient.EXPECT().GetStorageClass(gomock.Any(), validStorageClass).Return(nil, nil).AnyTimes()
+				namespaceStruct.Labels = map[string]string{"mutatevirtualmachines.kubemacpool.io": "allocate"}
```

Comment:
maybe you can move the map to be in generic part... it's the same map in all those test functions

Reply:
Defined the map as a variable